data/json/npcs/meteorologist.json

```diff
@@ -0,0 +1,311 @@
+              "time": "2 days"
+            }
+          }
+        },
+        "success": {
+          "effect": [ { "npc_lose_var": "meteorologist_wait", "type": "time", "context": "mission" } ],
+          "topic": "TALK_MISSION_LIST"
+        },
+        "failure": { "topic": "TALK_meteorologist_wait" }
+      },
+      { "text": "Goodbye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_meteorologist_wait",
+    "dynamic_line": "I need more time.  Come back later.",
+    "responses": [ { "text": "No problem.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_meteorologist_safety",
+    "dynamic_line": "Help?  Help with what?  Oh, yes, yes.  If you know a place with many survivors, can you take me there?  I doubt I can survive alone…",
+    "responses": [
+      {
+        "text": "Let's go.",
+        "topic": "TALK_DONE",
+        "effect": [ "follow_only", { "npc_first_topic": "TALK_meteorologist_safety_lead" } ]
+      },
+      { "text": "Want to travel with me?", "topic": "TALK_SUGGEST_FOLLOW" },
+      { "text": "Maybe later.", "topic": "TALK_NONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_meteorologist_safety_lead",
+    "dynamic_line": "Are we there yet, <name_g>?",
+    "responses": [
+      {
+        "text": "Yes, you can relax now.",
+        "topic": "TALK_meteorologist_safety_done",
+        "condition": {
+          "or": [
+            { "npc_at_om_location": "FACTION_CAMP_ANY" },
+            { "npc_at_om_location": "evac_center_18" },
+            { "npc_at_om_location": "robofachq_surface_entrance" }
+          ]
+        }
+      },
+      { "text": "Don't worry, we're getting there soon.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_meteorologist_safety_done",
+    "dynamic_line": "Thank you.  I will never forget your kindness, <name_g>.  Take this book as token of my gratitude.",
+    "responses": [
+      {
+        "text": "Bye.",
+        "topic": "TALK_DONE",
+        "effect": [ { "u_buy_item": "adv_chemistry" }, "stop_following", { "npc_first_topic": "TALK_meteorologist_safe" } ]
+      }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_meteorologist_safe",
+    "dynamic_line": "I'm glad you still alive.",
+    "responses": [ { "text": "Me too.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_meteorologist_Background",
+    "dynamic_line": "I'm just a regular meteorologist.  In recent years I have worked at meteostation, monitored the weather, state of air, read and wrote reports, drew up schedules…  In general, complete boredom.  In my free time I was studying electronics and radio engineering, making various devices, some for work, some just for fun.  When the first riots began, I gathered all my things that I managed to take with me and settled here.  Since then I've been sitting here, wasting my time doing all sorts of nonsense.",
```

Comment:
```suggestion
    "dynamic_line": "I'm just a regular meteorologist.  In recent years I have worked at weather station, monitored the weather, state of air, read and wrote reports, drew up schedules…  In general, complete boredom.  In my free time I was studying electronics and radio engineering, making various devices, some for work, some just for fun.  When the first riots began, I gathered all my things that I managed to take with me and settled here.  Since then I've been sitting here, wasting my time doing all sorts of nonsense.",
```